util/xgrpc/server.go

```diff
@@ -7,14 +7,17 @@ import (
 )
 
 func NewServer(logger *zap.Logger, extraOpts ...ServerOption) *grpc.Server {
-	opts := newOptions(logger, extraOpts...)
-	srv := grpc.NewServer(
-		append([]grpc.ServerOption{
-			grpc.RPCCompressor(grpc.NewGZIPCompressor()),
-			grpc.RPCDecompressor(grpc.NewGZIPDecompressor()),
-			grpc.UnaryInterceptor(grpc_middleware.ChainUnaryServer(opts.interceptors.u...)),
-			grpc.StreamInterceptor(grpc_middleware.ChainStreamServer(opts.interceptors.s...)),
-		}, opts.options...)...,
+	var (
```

Comment:
What is the secret knowledge replacing `:=` by `var`?

Reply:
It’s aesthetic!

Reply:
I think it's crap! Let's avoid doing unnecessary changes.